packages/editor-ui/src/components/ParameterOptions.vue

```diff
@@ -91,6 +95,18 @@ export default Vue.extend({
 			return !!this.getArgument('loadOptionsMethod') || !!this.getArgument('loadOptions');
 		},
 		actions(): Array<{ label: string; value: string; disabled?: boolean }> {
+			if (
+				this.ndvStore.activeNode?.type === HTML_NODE_TYPE &&
+				this.ndvStore.activeNode?.parameters.operation === 'generateHtmlTemplate'
+			) {
+				return [
+					{
+						label: 'Format HTML',
+						value: 'formatHtml',
+					},
+				];
+			}
+
```

Comment:
Thought about generalizing this as a node setting. Will refactor when setting up a component for use in params named `html` across all nodes.